### PR TITLE
Explain a bit more that lightning talks are signed up in person at the conf

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

**Comment by plaindocs** [comment]
Supercedes #745

**Review by plaindocs — APPROVED** [review]

File: docs/include/conf/events/lightning-talks.rst [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+At the beginning of each day, we will set a basket that you can put your proposal in.
+We will announce a cut off time,
+and after that we will pick from the submitted talks and announce who will be speaking that afternoon.
+
+We generally have two baskets for people who submit talks:
+
+* First-time speakers
+* Experienced speakers
+
+We do this so that we can prioritize people who are new to speaking,
```
Comment (mxsasha) [inline docs/include/conf/events/lightning-talks.rst]: We don't really prioritize for first-time speakers, right? We like to give them a fair chance, and maybe a bit more, but it's not like first-timers are always picked first. We often go for a mix.

Comment (plaindocs) [reply docs/include/conf/events/lightning-talks.rst]: Give first time speakers a fair chance?

Comment (plaindocs) [reply docs/include/conf/events/lightning-talks.rst]: Have a mix of first time speakers and experienced?

Comment (ericholscher) [reply docs/include/conf/events/lightning-talks.rst]: Pushed an update, let me know if it works.

**Pull request merged**